tests/integration/targets/autoscaling_group/roles/ec2_asg/tasks/create_update_delete.yml

```diff
@@ -474,8 +474,8 @@
 
     # Target group names have max length of 32 characters
   - set_fact:
-      tg1_name: "{{ (resource_prefix + '-tg1' ) | regex_search('(.{1,32})$') }}"
-      tg2_name: "{{ (resource_prefix + '-tg2' ) | regex_search('(.{1,32})$') }}"
+      tg1_name: "{{ ( resource_prefix | regex_search('^(.{1,28})') ) + '-tg1' }}"
+      tg2_name: "{{ ( resource_prefix | regex_search('^(.{1,28})') ) + '-tg2' }}"
```

Comment:
I know you're modifying what's there already, but it would be better to use "tiny_prefix" here (it didn't exist when the tests were first written)

See for example the elb_target tests - https://github.com/ansible-collections/community.aws/blob/main/tests/integration/targets/elb_target/defaults/main.yml

`"ansible-test-{{ tiny_prefix }}-asg-tg1"`  (I'd sneak in the asg here, it helps us track down the source when resources aren't cleaned up)

Reply:
I switched to tiny_prefix in https://github.com/ansible-collections/amazon.aws/pull/1593/commits/5fa66e3805dffe79dc9a50d7c4977dc8c093eb0b, did have to truncate it by one character to keep it under 32 chars.